catalyst-gateway/Earthfile

```
@@ -49,6 +49,10 @@ build:
     SAVE ARTIFACT target/doc doc
     SAVE ARTIFACT target/release/cat-gateway cat-gateway
 
+# build : Build the catalyst-gateway service. Used for nightly setup
+nightly:
+    FROM +build
```

```suggestion
   BUILD +build
```
You should use BUILD instead of FROM, because BUILD saves the outputs of the target (so the test report in this case)

---

updated